src/main/java/net/sf/jabref/importer/ImportFileFilter.java

```diff
@@ -42,11 +51,17 @@ public ImportFormat getImportFormat() {
 
     @Override
     public boolean accept(File file) {
-        return true;
-        /*if (file.isDirectory())
+        if (format.getExtensions() == null || format.getExtensions().isEmpty()) {
```

Comment:
Since almost no `ImportFormat` currently has this method implemented (asfaik only two have) it is.
If I remove this check this method will fail and won't update its view.